chat/chat-impl/impl/src/java/org/sakaiproject/chat2/model/impl/ChatEntityProducer.java

```diff
@@ -195,11 +194,20 @@ public String archive(String siteId, Document doc, Stack stack, String archivePa
     */
    public void archiveSynopticOptions(String siteId, Document doc, Element element)
    {
-      try
-      {
          // archive the synoptic tool options
-         Site site = siteService.getSite(siteId);
+         Site site;
+         try {
+            site = siteService.getSite(siteId);
+         } catch (IdUnusedException e) {
```

Comment:
Log?

Reply:
This is never going to happen, because this method is called from the archive service which by that time has already established that the site exists. So doesn't seem worth a log line.

Reply:
Steve while I agree with you a log line is super trivial here, I'll add it for you.